src/plots/Curve/CurveAttributes.code

```diff
@@ -253,11 +253,9 @@ Definition:
 //    Initialize colorIndex. 
 //
 // ****************************************************************************
-static int colorIndex;
 void
 CurveViewerEnginePluginInfo::InitializeGlobalObjects()
 {
-    colorIndex = 0;
```

Comment:
Since this code is no longer custom, it can be removed from the .code and .xml files.
Same with InitializePlotAtts.

Reply:
InitializePlotAtts still needs to be removed.